bin/yamlparse.py

```diff
@@ -0,0 +1,26 @@
+# This works with install-chart.sh to get the information to run the script.
+# It doesn't work very well as any kind of stand-alone utility.
+
+import sys
+import yaml
+
+if len(sys.argv) < 3:
+    print(f"Usage: {sys.argv[0]} <filename> <chart>")
+    sys.exit(1)
+
+filename = sys.argv[1]
+chart = sys.argv[2]
+
+with open(filename, "r") as conf_file:
+    conf_yaml = yaml.safe_load(conf_file)
```

Comment:
Maybe adding something like this would help us determine an invalid yaml file.
```suggestion
try:
    conf_yaml = yaml.safe_load(conf_file)
except Exception as e:
    print(f"Error parsing YAML file {filename}: {e}")
    sys.exit(1)
```

Reply:
added